cloud/docker/_docker.py

```diff
@@ -812,7 +812,7 @@ def __init__(self, module):
             use_tls = 'verify'
 
         tls_config = None
-        if use_tls != 'no':
+        if use_tls not in ('no', 'False'):
```

Comment:
`if use_tls not in BOOLEANS_FALSE` , you can import BOOLEANS_FALSE from basic.py in module_utils